### PR TITLE
Ford: Add Mach-E 

### PR DESCRIPTION
**Pull request by coffee-cake-isaac**
**Checklist**
- [x] added entry to CarInfo in selfdrive/car/*/values.py and ran `selfdrive/car/docs.py` to generate new docs
- [x] test route added to [routes.py](https://github.com/commaai/openpilot/blob/master/selfdrive/car/tests/routes.py)
- [x] route with openpilot:
- [ ] route with stock system:

No new `docs.py` due to the vehicle still being in dashcam mode.

### CODE REVIEW

File: selfdrive/car/ford/values.py [hunk 1]
```diff
@@ -79,6 +81,8 @@ class FordCarInfo(CarInfo):
   def init_make(self, CP: car.CarParams):
     if CP.carFingerprint in (CAR.BRONCO_SPORT_MK1, CAR.MAVERICK_MK1):
       self.car_parts = CarParts([Device.threex_angled_mount, CarHarness.ford_q3])
+    if CP.carFingerprint in (CAR.F_150_MK14, CAR.MUSTANG_MACH_E_MK1):
```
Comment (adeebshihadeh) [inline selfdrive/car/ford/values.py:84]:
```suggestion
    if CP.carFingerprint in CANFD_CAR:
```

**Comment by coffee-cake-isaac** [comment]
@sshane I did the PR break-out today instead of tomorrow since I had a bit of time. Will tag you in the F150 Lightning one as well.

**Review by adeebshihadeh — APPROVED** [review]
Also looks good once it's rebased. @sshane have you already verified all those FW versions?

**Comment by sshane** [comment]
no, haven't found time yet

**Comment by sshane** [comment]
Tried to push a rebase, but I still get 403 errors.

**Comment by sshane** [comment]
Only found two dongles in https://github.com/commaai/openpilot/pull/30444, checked and both engage, and ran one dongle through test_models, all good

**Comment by sshane** [comment]
thanks!

**Pull request merged**